docs.html

```diff
@@ -128,12 +130,119 @@ <h2 class="component-title">Button</h2>
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+            </section>
+
+            <!-- avatar -->
+            <section id="avatar">
+                <h2 class="component-title">Avatar</h2>
+                <p class="component-description">Avatar is used to show profile picture or initials of an user. Avatar
+                    is available in four sizes 'xsm', 'sm', 'md' and 'lg'.
+                </p>
+                <div class="avatar-output-container">
+                    <img class="avatar avatar-xsm" src="https://i.pravatar.cc/300">
+                    <img class="avatar avatar-sm" src="https://i.pravatar.cc/300">
+                    <img class="avatar avatar-md" src="https://i.pravatar.cc/300">
+                    <img class="avatar avatar-lg" src="https://i.pravatar.cc/300">
+                </div>
+                <iframe class="component-code"
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522avatar%2520avatar-xsm%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F300%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520avatar-sm%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F300%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520avatar-md%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F300%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520avatar-lg%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F300%2522%253E"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+                <p class="component-description">Four sizes of text Avatar are available in sizes 'xsm', 'sm', 'md' and 'lg'.
+                </p>
+                <div class="avatar-output-container">
+                    <div class="avatar avatar-xsm avatar-text avatar-text-xsm">VG</div>
+                    <div class="avatar avatar-sm avatar-text avatar-text-sm">VG</div>
+                    <div class="avatar avatar-md avatar-text avatar-text-md">VG</div>
```

Comment:
since you are using xsm sm md lg, then maybe you can discard the regular avatar-text class being used here.

Reply:
I looked into it, the base class is needed.